osidb/tests/test_sync_manager.py

```diff
@@ -0,0 +1,173 @@
+from datetime import datetime, timedelta, timezone
+from unittest.mock import Mock
+
+import pytest
+from celery.exceptions import Ignore
+from django.db import connection
+from django.db.models.base import ModelBase
+from django.test import TestCase
+from freezegun import freeze_time
+
+from osidb.sync_manager import (
+    JiraTaskDownloadManager,
+    JiraTaskSyncManager,
+    JiraTaskTransitionManager,
+    SyncManager,
+)
+from osidb.tests.factories import FlawFactory
+
+pytestmark = pytest.mark.unit
+
+
+class TestSyncManager(TestCase):
```

Comment:
I am consufed how does this work, django docs mentions `setUpClass`/`tearDownClass`  method but no `setup_class`/`teardown_class` https://docs.djangoproject.com/en/4.2/topics/testing/tools/#django.test.TestCase, haven't really worked much with django TestCases though :smile:

Reply:
FWIW, the setup/teardown can be achieved using pytest fixtures and we avoid mixing `pytest` and `unittest`.

Another alternative is defining the test model outside of the test's scope, I think we do this already for a few tests (so you can just reference the model directly and it only exists in the test database).